src/App.jsx

```diff
@@ -17,22 +18,33 @@ const tokenKey = 'token';
 
 const App = () => {
   const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const { isLoggedIn, role } = useSelector((state) => state.login);
 
   useEffect(() => {
     if (localStorage.getItem(tokenKey)) {
-      dispatch(login(JSON.parse(localStorage.getItem(tokenKey))));
+      dispatch(loginActions.login(JSON.parse(localStorage.getItem(tokenKey))));
     }
 
-    dispatch(getAllRoomsAsync());
+    dispatch(getAllCities());
   }, []);
 
+  useEffect(() => {
+    if (isLoggedIn) {
+      navigate('/', { replace: true });
+    }
+  }, [isLoggedIn]);
+
   const { isLoggedIn, role } = useSelector((state) => state.login);
```

Comment:
- [ ] This line is repeated in line 23, please fix this issue. This error is preventing netlify to build the app

Reply:
@williamrolando88 This is fixed